go.mod

```diff
@@ -4,30 +4,33 @@ go 1.13
 
 require (
 	github.com/Azure/azure-sdk-for-go v30.0.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.9.0
-	github.com/Azure/go-autorest/autorest/adal v0.5.0
+	github.com/Azure/go-autorest/autorest v0.9.6
+	github.com/Azure/go-autorest/autorest/adal v0.8.2
 	github.com/Azure/go-autorest/autorest/to v0.2.0 // indirect
 	github.com/Azure/go-autorest/autorest/validation v0.1.0 // indirect
 	github.com/aws/aws-sdk-go v1.15.72
 	github.com/davecgh/go-spew v1.1.1
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-logr/logr v0.1.0
-	github.com/go-logr/zapr v0.1.1
-	github.com/google/go-cmp v0.3.1
+	github.com/go-logr/logr v0.2.0
+	github.com/go-logr/zapr v0.2.0
+	github.com/google/go-cmp v0.4.0
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
 	github.com/openshift/api v0.0.0-20200609191024-dca637550e8c
```

Comment:
Should this PR also bump the vendored version of `openshift/api`?

Reply:
`openshift/api` bump is not required for this PR. `openshift/api` will be bumped in https://github.com/openshift/cluster-ingress-operator/pull/427